File: unity/Assets/Scripts/Multiplayer/Rooms/RoomLeaveResponse.cs
using Michitai.Multiplayer.Errors;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Michitai.Multiplayer.Rooms
{
    /// <summary>
    /// Response returned when a player successfully leaves a game room.
    /// </summary>
    [System.Serializable]
    public class RoomLeaveResponse : ApiResponse<ERoomLeaveError>
    {
        /// <summary>
        /// Confirmation message for leaving the room.
        /// </summary>
        public string message;
    }
}

File: java/src/main/java/com/michitai/multiplayer/rooms/RoomLeaveResponse.java
package com.michitai.multiplayer.rooms;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.michitai.multiplayer.ApiResponse;

/**
 * Response returned when a player successfully leaves a game room.
 */
public class RoomLeaveResponse extends ApiResponse {
    @JsonProperty("message")
    private String message;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
